substrate-archive/src/migrations/20211108182913_extrinsics.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS extrinsics (
+	id SERIAL NOT NULL,
+	hash bytea NOT NULL PRIMARY KEY,
+	number int check (number >= 0 and number < 2147483647) NOT NULL UNIQUE,
```

Comment:
I guess the `< 2147483647` is not necessary since int is signed 4 bytes, but there's something nice about documenting the range explicitly anyway, since you need the `>=0` bit :)

Reply:
Second Q: Should `number` be a foreign key pointing to `blocks.block_num`?

Reply:
That's fair! Def happy with it merging as-is for now :)